src/main/java/Duke.java

```diff
@@ -1,10 +1,151 @@
+import java.util.Scanner;
 public class Duke {
+    public static final int MAX_TASKS = 100;
+    private static int taskCount = 0;
+    static Task[] tasks = new Task[MAX_TASKS];
```

Comment:
Absence of magic numbers by explaining meaning of number in the variable name. Follows code readability standards. Good job!